### DISCUSSION TITLE
How to edit a single address? (UK)

### DISCUSSION THREAD

**Discussion by Keoth24** [osm-search/Nominatim#3550] [Q&A]
How can I get this updates please as my deliveries are not getting to me 
![IMG_1249](https://github.com/user-attachments/assets/3f3675ac-5185-4aa9-a2b5-41b77ea5925a)

**Comment by mtmail** [comment]
Hi @Keoth24 

This seems to be building: https://www.openstreetmap.org/way/235061082 On the top of the page you'll see an 'Edit' button. Similar to wikipedia you can edit the data yourself. After clicking you'll see fields for the address: street, housenumber etc.

While some services update after a few minutes, others need 24-48 hours. I have no idea who the lpl5 website or app from your (other) screenshot is. Or how often they update their data. I hope you also contacted your delivery company to sort this out, too, they're your service provider responsible for delivering your food(?), not openstreetmap.

![image](https://github.com/user-attachments/assets/d124f73a-e773-435a-b1cc-b0caa6e7dd1a)
